src/views/Order.vue

```diff
@@ -44,34 +47,35 @@
                 <ion-icon slot="end" :icon="informationCircleOutline" />
               </ion-item>
             </ion-list>
-          </div>
+          </div> -->
 
           <div class="info">
             <ion-card>
               <ion-list>
                 <ion-item lines="none">
-                  <ion-label>John Doe</ion-label>
-                  <ion-chip slot="end">
+                  <ion-label> {{ order.customer?.name }} </ion-label>
+                  <!-- TODO: handle this property to display loyalty options -->
+                  <ion-chip slot="end" v-if="$filters.getCustomerLoyalty(order.orderNotes, cusotmerLoyaltyOptions)">
```

Comment:
I think we can have the same approach as identifications for customer loyalty as well.
WDYT?

Reply:
Ok Sir, will create a helper for the same.

Reply:
Sure Sir